### ISSUE TITLE
Replace Trello with GitHub Projects in "How to Help Translate freeCodeCamp into Your Native Language" article

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#54436]
### Describe the Issue

We have migrated to from Trello to GitHub projects so this section should be updated:

<img width="802" alt="Screenshot 2024-04-18 at 12 43 03" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/25715018/90b30eaa-12f5-41cf-bdf3-7cdd2933cc2a">

### Affected Page

https://www.freecodecamp.org/news/help-translate-freecodecamp-language/

**Comment by scissorsneedfoodtoo** [comment]
Thanks for your patience all. I'll figure out the current flow for onboaring new translators and will update this article.

**Comment by RafaelDavisH** [comment]
We have our article translations onboarding process on our repo, "news-translations-tasks" README. 
Link: https://github.com/freeCodeCamp/news-translation-tasks

Here are two screenshots that can be of use to replace the current Trello board screenshot. 

It's important to state that we only have 7 GH project currently active, and there is no date set to when we would open up for the rest of the world languages. It's one of our priorities, to make sure more world languages have the opportunity to localized freeCodeCamp top English articles/handbooks to their languages. 

![Screenshot 2024-05-30 at 9 37 44 AM](https://github.com/freeCodeCamp/freeCodeCamp/assets/6822714/db19db35-3c8d-4ff9-9a58-27bbdc6620e1)
![Screenshot 2024-05-30 at 9 35 08 AM](https://github.com/freeCodeCamp/freeCodeCamp/assets/6822714/ef6f716e-54db-45c3-abd4-4e67199ba324)

**Comment by scissorsneedfoodtoo** [comment]
Thanks again for sharing those details here, @RafaelDavisH.

I went in and updated the screenshots and wording slightly for the current onboarding process. I tried to keep a light touch so it still sounds like Quincy's voice while reading.

To avoid coming off as too negative, I didn't explicitly mention that we currently have just 7 project board for the languages we're actively translating News articles into. Eventually we'll add more languages, and I figure that contributors can still get involved somehow even if we don't currently have a board for their native language.

The updates should go live within the next few hours after the next build.
